spec/models/solr_document_spec.rb

```diff
@@ -124,6 +124,33 @@
     end
   end
 
+  describe 'check ersources location_display' do
+    solr_info = YAML.load_file(Rails.root.join('spec/fixtures/solr_documents/30.yml'))
```

Comment:
We usually reserve fixtures for feature tests.  If we don't have a feature test here, it might just be easier to instantiate a `SolrDocument` inline here and skip the fixture.  Adding fixtures is something we try to be judicious with since they make the repository big and it's a bunch of indirection when you're trying to figure out a test.